documentation/guides/release-and-deploy.md

```diff
@@ -49,3 +49,17 @@ git push origin master --follow-tags
 - Log in to [Shipit](https://shipit.shopify.io/shopify/quilt/production)
 
 - When CI is 🍏 on the commit titled `Publish`, press `Deploy` to update packages on npm.
+
+# Beta releases for 🎩ing changes?
+
+### Step 1 - publish a beta(s)
+
+- In your branch, run `yarn run release-beta`. Lerna will launch it's CLI to select a version for the changed packages. Select the `Custom` option and enter a version with an appended `-beta.X` (eg. `0.29.10-my-feature-beta.1`)
```

Comment:
This could maybe have some instructions on what to do when a package change causes other packages to be bumped as well. 🙂